public/docs/_examples/intro/dart/lib/hero_detail_component.dart

```diff
@@ -0,0 +1,10 @@
+library developer_guide_intro.hero_detail_component;
+
+import 'package:angular2/angular2.dart';
+import 'package:developer_guide_intro/hero.dart';
+
+@Component(selector: 'hero-detail', templateUrl: 'hero_detail_component.html')
```

Comment:
No directives field is necessary? (I'm comparing this to https://github.com/tomplusplus/angular.io/blob/DeveloperGuides/intro/public/docs/_examples/intro/ts/src/app/hero-detail.component.ts)